src/main/webapp/WEB-INF/view/conversations.jsp

```diff
@@ -45,13 +46,34 @@
     <% } %>
 
     <% if(request.getSession().getAttribute("user") != null){ %>
-      <h1>New Conversation</h1>
+      <h1>Create Group Conversation</h1>
       <form action="/conversations" method="POST">
           <div class="form-group">
-            <label class="form-control-label">Title:</label>
+            <label class="form-control-label">Group Name:</label>
           <input type="text" name="conversationTitle">
-        </div>
+          </div>
+          <div class="form-group">
+            <label for="AccessControl">Set Access</label>
+            <select name="AccessControl" style="width:300px;">
+              <option value="Public">Public</option>
+              <option value="Private">Private</option>
+            </select>
+          </div>
+          <h4>(Only if Conversation is private)</h4>
+          <div class="form-group"
+            <label for="Userlabel">Add Users</label>
+            <% List<User> users =(List<User>) request.getAttribute("ConvoUsers");%>
+            <select name="UserLabel" style="width:300px;">
+               <option value="add user">add user</option>
```

Comment:
I think we no longer want this place holder "add user" value, since all of the values would be the users

Reply:
This was to serve as a default value

Reply:
but I can take it out if you want

Reply:
The first answer on this stackoverflow has a good example of how we could do this: https://stackoverflow.com/questions/8605516/default-select-option-as-blank/45130085